src/commands/createFunction/FunctionSubWizard.ts

```diff
@@ -9,7 +9,7 @@ import { canValidateAzureWebJobStorageOnDebug } from '../../debug/validatePreDeb
 import { getAzureWebJobsStorage } from '../../funcConfig/local.settings';
```

Comment:
Can we hide the OpenAPI one at the bottom for the new model? I'm almost certain it's not compatible

<img width="486" alt="Screen Shot 2022-12-01 at 4 23 42 PM" src="https://user-images.githubusercontent.com/11282622/205186920-61be7eff-c0dd-4ba9-bb08-d0b00c982dbd.png">